logger.js

```diff
@@ -0,0 +1,20 @@
+const winston = require('winston');
+
+const level = process.env.LOG_LEVEL || 'debug';
+
+const logger = winston.createLogger({
+  transports: [
+    new winston.transports.Console({
+      format: winston.format.combine(
+        winston.format.colorize(),
+        winston.format.simple()
+      ),
+      level: level,
+      timestamp: function() {
+        return new Date().toISOString();
+      }
```

Comment:
I'm not seeing the timestamp in the log output:
```
info: SummaryLink Creation is: https://api.smmry.com?SM_API_KEY=28555DB748&SM_URL=
info: User-Agent is: PostmanRuntime/7.1.1
info: Command = ScoutTitles
```

Reply:
I think we don't need timestamps in the current implementation (console output) as AWS already provides them. 
```
2018-06-27T22:30:43.052124+00:00 app[web.1]: info: 	User-Agent is: PostmanRuntime/7.1.5
2018-06-27T22:30:43.052304+00:00 app[web.1]: info: 	Command = ScoutTitles
```

If we implement another form of logging (such as files) we should add the timestamp (and actually we just have to add `winston.format.timestamp()` to the formatting). But I think for console logging it just adds unnecessary text to the output. I added a commit to remove that. If you approve, I'll also remove the commit form the scout-alexa logging PR.